### PR TITLE
Feature: Add support for BitBucket Repository Access Token

### PR DESCRIPTION
**Pull request by thawankeane**
BitBucket supports [repositories access tokens](https://support.atlassian.com/bitbucket-cloud/docs/repository-access-tokens/), these tokens can access the repository through scopes, it can be used to authenticate with BitBucket REST API to make changes to repo itself.

This pull request add support for them through a new environment variable called `DANGER_BITBUCKETCLOUD_REPO_ACCESSTOKEN`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 <!-- Your comment below this -->
 
+- Add support for BitBucket Cloud Repository Access Token - [@thawankeane]
```
Comment (glensc) [inline CHANGELOG.md:19]: did you read the preamble of changelog? you need to add your handle link bottom of the file. see the misrendering yourself:
- https://github.com/thawankeane/danger-js/blob/3ce2eae0c58311b72ae787c3af0501c70fd610af/CHANGELOG.md

Comment (thawankeane) [reply CHANGELOG.md:19]: Oops! I didn't note it, sorry. Fixing it right now

**Comment by orta** [comment]
A test to make sure it doesn't  break in the future would be nice, but it's not a blocker (that can come in another PR if you're in the mood) but this looks good to me - thanks!

**Comment by orta** [comment]
Thanks, this is shipped as 11.2.4

**Pull request merged**